task/04-arrays-tasks.js

```diff
@@ -559,8 +632,16 @@ function getElementByIndexes(arr, indexes) {
  * 
  */
 function swapHeadAndTail(arr) {
-    throw new Error('Not implemented');
-}
+   const middleIndex = Math.floor(arr.length / 2);
+   let newStart = arr.slice(middleIndex);
+   let newEnd = arr.slice(0, middleIndex);
+
+   if (arr.length % 2 === 0) {
+      return newStart.concat(newEnd);
+   } else {
```

Comment:
You can return result without `else` operator because you returns result in `if`.